components/Avatar/avatar.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lazy Avatar</title>
+    <link rel="stylesheet" href="/style.css" />
+
+    <link rel="stylesheet" href="/prism/prism.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="navbar">
+        <!-- <span class="navbar__logo"> </span> -->
+        <a href="/index.html" class="navbar__logo">LazyCss</a>
+
+        <ul class="navbar__list">
+          <li><a href="/index.html">Home </a></li>
+          <li><a href="#">Docs </a></li>
+          <li>
+            <a
+              href="https://github.com/KuldeepSinghRathore/lazycss/tree/developement"
+              >Github
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="grid--section">
+      <aside class="aside">
+        <ul class="aside__list">
+          <li class="component-links">
+            <a href="/components/Button/button.html">Button</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Avatar/avatar.html">Avatar</a>
+          </li>
+          <li class="component-links">
+            <a href="/components/Alert/alert.html">Alert</a>
```

Comment:
Here try to be consistent while naming the class names keep any one of these '-' or '--' or '_' or  '__' , don't try to use all of these . I have seen this mistake in many places in code (check this one)